src/config/webpack/webpack.config.js

```diff
@@ -116,7 +116,7 @@ module.exports = (CONFIG, SWANKY_CONFIG) => {
           use: [{
             loader: 'babel-loader',
             query: {
-              presets: ['es2015']
+              presets: ['es2015', 'react']
```

Comment:
I don't believe it will. I'm using it in a test project which is rendering React and Angular components (on different pages) at the moment.